src/main/java/org/folio/service/event/EntityChangedEventPublisherFactory.java

```diff
@@ -56,4 +61,17 @@ public static EntityChangedEventPublisher<String, CheckIn> checkInEventPublisher
         new CheckInRepository(vertxContext, okapiHeaders));
   }
 
+  public static EntityChangedEventPublisher<String, CirculationRules> circulationRulesEventPublisher(
+    Context vertxContext, Map<String, String> okapiHeaders) {
+
+    log.info("circulationRulesEventPublisher:: creating circulation rules event publisher");
+
+    return new EntityChangedEventPublisher<>(okapiHeaders, CirculationRules::getId, NULL_ID,
+      new EntityChangedEventFactory<>(),
+      new DomainEventPublisher<>(vertxContext,
+        RULES.fullTopicName(tenantId(okapiHeaders)),
+        FailureHandler.noOperation()),
+      new CirculationRulesRepository(vertxContext, okapiHeaders));
```

Comment:
```suggestion
      new EntityChangedEventFactory<>(), new DomainEventPublisher<>(vertxContext,
        RULES.fullTopicName(tenantId(okapiHeaders)), FailureHandler.noOperation()),
      new CirculationRulesRepository(vertxContext, okapiHeaders));
```

Reply:
All initializations in this class are done in this style. And I think in this case it actually makes things a bit more readable.